pyproject.toml

```diff
@@ -18,7 +18,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7"
 mutagen = "^1.44"
-feedparser = "^5.2"
+feedparser = "^5.2 || ^6.0"
```

Comment:
woah, you can do that? What does it mean - choose whichever works?

Reply:
good question, there is only one case documented for specifying the Python version: https://python-poetry.org/docs/dependency-specification/#multiple-constraints-dependencies

It means either of those constrains with npm/composer, and poetry seems to understand it as well.

Reply:
TIL, thanks :)

Reply:
Just checked: When you create a dist with `poetry build` it converts it to `feedparser>=5.2,<7.0` in the generated setup.py.